src/raygun.rum.js

```diff
@@ -873,6 +895,36 @@ var raygunRumFactory = function(window, $, Raygun) {
     // =                                                                              =
     // ================================================================================
 
+    /**
+     * Returns true if the resources entry type is set to "measure"
+     */
+    function isCustomTimingMeasurement(resource) {
+      return resource && resource.entryType === "measure";
+    }    
+
+    /**
+     * Creates a custom timing measurement from a ResourceMeasure value passed.
+     * The ResourceMeasure object passed in should be retrieved from the window.performance API
+     */
+    function getCustomTimingMeasurement(resource) {
+      return createCustomTimingMeasurement(resource.name, resource.duration, resource.startTime);
+    }
+
+    /**
+     * Creates a custom timing measurement for a name and duration passed.
+     * This can be used to create custom timings seperate to the window.performance API
```

Comment:
Nit: should be 'separate' as 'seperate' is a common misspelling

Reply:
Fixed in a seperate PR